doc/source/tune-usage.rst

```diff
@@ -296,6 +296,40 @@ of a trial, you can additionally set the checkpoint_at_end to True. An example i
         },
     })
 
+Recovering From Failures (Experimental)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
```suggestion
 Tune automatically persists the progress of your experiments, so if an experiment crashes or is otherwise cancelled, it can be resumed by passing `resume=True` in `run_experiments()`. If resume is not enabled, a new experiment will be created instead. You can always force a new experiment to be created by changing the experiment name.

Note that trials will be restored to their last checkpoint. If checkpointing is not enabled, unfinished trials will be restarted from scratch.

Reply:
(this replaces the current copy)